README.md

```diff
@@ -1,79 +1,134 @@
+$ hyperdash demo
 ```
 
-or
+## Learn Hyperdash in 30 seconds
+The core object of Hyperdash is the **Experiment**. The simplest experiment logs a single print statement.
+```python
+# simple.py
+from hyperdash import Experiment
+
+# Create an experiment with a model name and autostart
+exp = Experiment("Print Example")
+
+print("View me on web or mobile")
 
+# cleanup
+exp.end()
 ```
-hyperdash run -n "My test bash script" ./my_test_bash_script.sh
+Running `python simple.py` causes a log of all the console output between experiment creation and end to be logged to local disk. For example:
+```
+View me on web or mobile
+Logs for this run of Print Example are available locally at: /Users/username/.hyperdash/logs/print-example/print-example_2017-09-16t23-00-25-833357.log
 ```
+### Instrumentation example
+Hyperdash helps you track **hyperparameters** and **performance metrics** for your experiments.
+```python
+# digits.py
+from sklearn import svm, datasets
+from hyperdash import Experiment
 
-It doesn't matter what language your script is written in, if it can be executed from the command line then you can wrap it with the `run`  command
+exp = Experiment("Digits Classifier")
 
-### Decorating a Python function
-Import the monitor function, and apply it as a decorator to a function that runs your machine learning job. The only argument you need to pass to the monitor function is the name of the model that you're training.
+# Preprocess data
+digits = datasets.load_digits()
+test_cases = 50
+X_train, y_train = digits.data[:-test_cases], digits.target[:-test_cases]
+X_test, y_test = digits.data[-test_cases:], digits.target[-test_cases:]
 
-```
-# test_script.py
+# Record the value of hyperparameter gamma for this experiment
+gamma = exp.param("gamma", 0.1)
+# Param can record any basic type (Number, Boolean, String)
 
-from hyperdash.sdk import monitor
+classifer = svm.SVC(gamma=gamma)
+classifer.fit(X_train, y_train)
 
-@monitor("dogs vs. cats")
-def train_dogs_vs_cats():
-  print("Epoch 1, accuracy: 50%")
-  time.sleep(2)
-  print("Epoch 2, accuracy: 75%")
-  time.sleep(2)
-  print("Epoch 3, accuracy: 100%")
+# Record a numerical performance metric
+exp.metric("accuracy", classifer.score(X_test, y_test))
+
+exp.end()
 ```
+Hyperparameters and metrics are pretty printed for your logs and reference:
+```
+{ gamma     : 0.001  }
+| accuracy  : 1.000  |
+Logs for this run of Digits Classifier are available locally at: /Users/username/.hyperdash/logs/digits-classifier/digits-classifier_2017-09-20t18-50-55-258215.log
+```
+**Done!**
+
+Now you can visualize your experiments in the Hyperdash [__web__](https://hyperdash.io/dashboard), [__iOS__](https://itunes.apple.com/us/app/hyperdash-machine-learning-monitoring/id1257582233), and [__Android__](https://play.google.com/store/apps/details?id=com.hyperdash) apps.
 
-Once you've imported our library, run your program as you normally would:
+### IPython/Jupyter Notebook
```

Comment:
Same comment as before. Tell them if they just want to track a cell really quickly, add this one line of code, but alternatively both the decorator and experiment API also work in Jupyter